ext/semian/sysv_semaphores.c

```diff
@@ -63,9 +63,23 @@ initialize_semaphore_set(semian_resource_t* res, const char* id_str, long permis
     rb_raise(eInternal, "error incrementing registered workers, errno: %d (%s)", errno, strerror(errno));
   }
 
+  int state = 0;
   sem_meta_lock(res->sem_id); // Sets otime for the first time by acquiring the sem lock
-  configure_tickets(res->sem_id, tickets,  quota);
+
+  configure_tickets_args_t configure_tickets_args = (configure_tickets_args_t){
+    .sem_id = res->sem_id,
+    .tickets = tickets,
+    .quota = quota,
+  };
+  rb_protect(
```

Comment:
[As far as I can gather](https://silverhammermba.github.io/emberb/c/#rb_protect) this is required because `configure_tickets` might raise, which has to be done within a `rb_protect` call to not segfault the VM?